Src/IronPythonConsoleAny/IronPythonConsoleAny.csproj

```diff
@@ -1,8 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$(ProjectDir)..\..\Build\Common.proj" />
   <PropertyGroup>
-    <TargetFrameworks>net40;net45</TargetFrameworks>
-    <TargetFrameworks Condition="'$(BuildNetCoreApp)' == 'true'">$(TargetFrameworks);netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(TargetFrameworks)' == ''">net40;net45</TargetFrameworks>
```

Comment:
Missing netcoreapp2.0.

Reply:
I didn't think we needed to build the console for netcoreapp2.0. Do we?